parallaxinc/Parallax-IDE
# Progressbar

#### What's this PR do?

Adds a determinate progress bar to the download screen in ChromeIDE. Shows progress as code is downloaded to the device, as required in Issue #82.
#### Where should the reviewer start?

Connect a BASIC Stamp to your computer and create a valid test file with working code.
#### How should this be manually tested?

With your valid code file, open the menu, click the download button. Wait for your device to load. Click the device. Then click Download to send code to the device. You should see the progress bar quickly slide across the bottom of the screen.
#### Any background context you want to provide?

No
#### What are the relevant tickets?

Issue #82 
#### Screenshots (if appropriate)

![image](https://cloud.githubusercontent.com/assets/2586086/7757304/1d99cd02-ffb6-11e4-9f2a-b50ddfce4972.png)
#### Questions:
- Is there a blog post?
  No
- Does the knowledge base need an update?
  No
